core/src/extcmd/command_request.cc

```diff
@@ -17,7 +17,7 @@
 */
 
 #include <cstdlib>
-#include <QUuid>
+#include "com/centreon/broker/misc/misc.hh"
```

Comment:
Then we would have com/centreon/broker/misc/uuid.hh which is more explicit.

Reply:
done